client/src/pages/rollup/Show.js

```diff
@@ -567,8 +576,15 @@ const RollupShow = ({ pageDispatchers, searchQuery, setSearchQuery }) => {
   }
 
   function changePeriod(nextPeriod) {
-    const periodStart = moment(queryParams.releasedAtStart).startOf(nextPeriod)
-    const periodEnd = moment(queryParams.releasedAtStart).endOf(nextPeriod)
+    const periodStart =
+      nextPeriod === PERIOD_6_MONTHS
+        ? moment(queryParams.releasedAtStart).startOf(QUARTER)
```

Comment:
I would expect half years to always be from Jan 1 - Jun 30 and Jul 1 - Dec 31, but with this logic they can shift by 3 months, which is unwanted.

Reply:
You should also update the `showNextPeriod()` and `showPreviousPeriod()` functions.